playwright/README.md

```diff
@@ -227,6 +227,87 @@ new AxeBuilder(page)
         .setLegacyMode(true);
 ```
 
+## Limit Frame Testing
+
+Including or excluding specific sections within a frame can be done with a `FromFrames` selector object.
+
+### AxeBuilder#include(FromFrames fromFrames)
+
+The following shows how to test all `form` elements a `#paymentFrame` frame or iframe:
+
+```java
+
+// Test each <form> inside each #paymentFrame frame or iframe:
+new AxeBuilder()
+        .include(new FromFrames("#paymentFrame", "form"));
```

Comment:
Should provide the import code for where `FromFrames` comes from

Reply:
Good catch!